data/registry/tools-cpp-conan.yml

```
@@ -0,0 +1,13 @@
+title: Conan Package for OpenTelemetry C++
+registryType: core
+isThirdParty: true
+language: cpp
+tags:
+  - cpp
+  - conan
+  - package
+repo: https://conan.io/center/opentelemetry-cpp
+license: MIT
+description: If you are using Conan to manage your dependencies, add opentelemetry-cpp/x.y.z to your conanfile's requires.
+authors: Others
```

This sets a precedent. I'd rather leave it blank (or put unknown):
```suggestion
authors:
```